### ISSUE TITLE
Bug: selectizeInput allows an empty value to be selected

### ISSUE THREAD

**Issue by daattali** [rstudio/shiny#3727]
```r
selectInput("test", "test", "1")
```

vs

```r
selectizeInput("test", "test", "1")
```

The first one must have a value, the second value allows the user to press *backspace* and be empty. This is not documented and as far as I can tell it's an unintended bug.

**Comment by daattali** [comment]
Are issues in this repo being tracked? I've opened 6 issues since November and there has been radio silence, which is very unsual. I've opened almost 150 issues in this repo and almost always there's either a comment or triage or something to indicate that it's being looked at. I hope this isn't because py-shiny is taking dev attention away :p

**Comment by cpsievert** [comment]
Yes, you bring up some good points in these issues, but they aren't currently at the top of our priority list (we're currently busy releasing `{shiny}`, `{bslib}`, `{htmlwidgets}`, `{htmltools}`, and `{httpuv}`, so we're certainly still devoting time to r-shiny development :)

Also, this is the sort of issue (as well as other issues like https://github.com/rstudio/shiny/issues/3738) where we probably wouldn't invest the time to change the existing behavior (even though it's sub-optimal) since there are almost certainly apps out there using this "bug" as a "feature".

**Comment by daattali** [comment]
> Also, this is the sort of issue (as well as other issues like https://github.com/rstudio/shiny/issues/3738) where we probably wouldn't invest the time to change the existing behavior (even though it's sub-optimal) since there are almost certainly apps out there using this "bug" as a "feature".

Good point, maybe instead of fixing it, there could be an addition boolean parameter specifying whether or not choosing empty is allowed, with TRUE being the default since that was the behaviour until now.
